oem/ibm/libpldmresponder/file_io_type_dump.cpp

```diff
@@ -99,17 +99,17 @@ std::string DumpHandler::findDumpObjPath(uint32_t fileHandle)
     }
 
     dbus::ObjectValueTree objects;
-    auto method =
-        bus.new_method_call(DUMP_MANAGER_BUSNAME, DUMP_MANAGER_PATH,
-                            OBJECT_MANAGER_INTERFACE, "GetManagedObjects");
 
     try
     {
+        auto method =
+            bus.new_method_call(DUMP_MANAGER_BUSNAME, DUMP_MANAGER_PATH,
+                                OBJECT_MANAGER_INTERFACE, "GetManagedObjects");
         auto reply = bus.call(method);
         reply.read(objects);
     }
 
-    catch (const std::exception& e)
+    catch (const sdbusplus::exception_t& e)
```

Comment:
could you also make this change to other catch blocks in the file as well?

Reply:
done